src/Parser/ParserErrorsException.php

```diff
@@ -22,7 +22,7 @@ public function __construct(
 		private ?string $parsedFile,
 	)
 	{
-		parent::__construct(implode(', ', array_map(static fn (Error $error): string => $error->getMessage(), $errors)));
+		parent::__construct(implode(', ', array_map(static fn (Error $error): string => $error->getRawMessage(), $errors)));
```

Comment:
I think one of the places are not tested. Please see the path where this is actually used, perhaps by totally breaking this piece of code and seeing where it fails.

Reply:
`src/Analyser/FileAnalyser.php` change is currently tested by `tests/PHPStan/Analyser/AnalyserIntegrationTest.php::testBug10358`

Reply:
The other exception is consumed in https://github.com/phpstan/phpstan-src/blob/1.10.67/src/Analyser/FileAnalyser.php#L211 but the message is not used - so there is currently not easy way to test it.